kitsune/wiki/tasks.py

```diff
@@ -166,7 +166,7 @@ def add_short_links(doc_ids):
         unpin_this_thread()
 
 
-@task(rate_limit='3/h')
+@task(rate_limit='3/h', priority=9)
```

Comment:
This task shouldn't have such a low priority

Reply:
From anecdotal evidence when inspecting the queue on dev, this seems to be the task that clogs everything else up. But I'll remove the priority for now and we can reconsider when we have proper numbers coming through from flower.